apply/cloud_applier.go

```diff
@@ -35,13 +35,42 @@ func NewAliCloudProvider(cluster *v1.Cluster) Interface {
 	return &CloudApplier{d}
 }
 
+func (c *CloudApplier) ScaleDownNodes(cluster *v1.Cluster) (isScaleDown bool, err error) {
```

Comment:
This is a ScaleDownNodes func, but it update c.NodesToJoin within the func, feel confused

Reply:
Oh, we needn't update it, I will fixed it